src/handler_base.js

```diff
@@ -170,6 +170,15 @@ export default class HandlerBase extends EventEmitter {
         }
     }
 
+    getStats() {
+        return {
+            srcTxBytes: this.srcSocket ? this.srcSocket.bytesWritten : undefined,
+            srcRxBytes: this.srcSocket ? this.srcSocket.bytesRead : undefined,
+            trgTxBytes: this.trgSocket ? this.trgSocket.bytesWritten : undefined,
+            trgRxBytes: this.trgSocket ? this.trgSocket.bytesRead : undefined,
+        };
+    }
+
```

Comment:
please change undefined to null, it's more transparent (e.g. if you call JSON.stringify)